src/main/java/org/jabref/logic/journals/JournalAbbreviationLoader.java

```diff
@@ -35,13 +28,6 @@
         return parser.getAbbreviations();
     }
 
-    private static List<Abbreviation> readJournalListFromFile(Path file, Charset encoding) throws IOException {
```

Comment:
With this method removed, how are custom journal abbreviation lists (csv files) loaded?

Reply:
This specific overload was never used locally. The